### PR TITLE
Removed likes and comments and added inline in bottom - fixed tslint

### LINKED ISSUE #97: Rething the display of likes and comments on full image view
We should try to move the numbers (how many likes and comments) next to the heart icon and the comment icon?    Makes no sense like it is right now.

45745601_2240977436179991_7902717718443327488_n

### PR DESCRIPTION
**Pull request by alexcroitoru95**
Fixes #97.

### CODE REVIEW

File: packages/RNSocialX/src/components/interaction/MediaInteractionButtons.style.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { Row } from 'native-base';
```
Comment (alexsirbu04) [inline packages/RNSocialX/src/components/interaction/MediaInteractionButtons.style.ts]: ?

File: packages/RNSocialX/src/components/displayers/MediaObjectViewer.tsx [hunk 2]
```diff
@@ -54,53 +54,49 @@ export const MediaObjectViewer: React.SFC<IMediaObjectViewerProps> = ({
...
-						onPressVideo={onPress}
-						containerStyle={customStyle}
-						muted={muted}
-						thumbOnly={thumbOnly}
-						resizeMode={resizeMode}
-						resizeToChangeAspectRatio={resizeToChangeAspectRatio}
-						paused={paused}
-					/>
-				)}
+			{mediaMimeType && mediaMimeType.startsWith(MediaTypeImage.key) && (
```
Comment (Shadyzpop) [inline packages/RNSocialX/src/components/displayers/MediaObjectViewer.tsx]: for android purposes
```suggestion
			{!!mediaMimeType && mediaMimeType.startsWith(MediaTypeImage.key) && (
```

File: packages/RNSocialX/src/components/interaction/MediaInteractionButtons.style.ts (hunk 1 above)
Comment (alexcroitoru95) [reply packages/RNSocialX/src/components/interaction/MediaInteractionButtons.style.ts]: I think Ionut imported it, I'll look into it

File: packages/RNSocialX/src/components/displayers/MediaObjectViewer.tsx (hunk 2 above)
Comment (alexcroitoru95) [reply packages/RNSocialX/src/components/displayers/MediaObjectViewer.tsx]: Done 🥇

**Pull request merged**